### PR TITLE
MGDCTRS-1477 feat: improve loading cards

### PR DESCRIPTION
**Pull request by gashcrumb**
This change swaps out the loading spinner for a card containing skeleton elements, and ensures that for either the list view or the card view that the card sizes are at fixed sizes to optimize the viewport scrolling.  Also this change brings in the latest patternfly version to gain access to the new source/sink icons.

And a few layout tweaks after some feedback.

Screenshot from 2022-11-14 13-56-37

### CODE REVIEW

**Comment by gashcrumb** [comment]
Mostly getting this in to make it available to the designers in chromatic, but I wouldn't mind a double-check that bumping the pf dependency doesn't cause any problems.

**Review by indraraj — APPROVED** [review]
LGTM

**Comment by indraraj** [comment]
Maybe we can defer merging this until today's prod deployment. @gashcrumb

File: src/ProofOfConcepts/ConnectorTypesGallery/ConnectorTypesGallery.stories.tsx [hunk 1]
```diff
@@ -33,21 +38,21 @@ const Template: ComponentStory<typeof ConnectorTypesGalleryWrapper> = (
...
+  'category-integration-and-delivery',
+  'category-logging',
+  'category-monitoring',
+  'category-crm',
+  'category-storage',
+  'category-streaming-and-messaging',
+  'category-amazon',
+  'category-google',
+  'category-azure',
 ];
```
Comment (gashcrumb) [inline src/ProofOfConcepts/ConnectorTypesGallery/ConnectorTypesGallery.stories.tsx:56]: @dhirajsb how does this look for category label values?  These become keys to the entries in the i18n file.

Comment (dhirajsb) [reply src/ProofOfConcepts/ConnectorTypesGallery/ConnectorTypesGallery.stories.tsx:56]: lgtm

**Pull request merged**